filer/admin/imageadmin.py

```diff
@@ -50,6 +50,10 @@ def clean_subject_location(self):
             # if supplied subject location is empty, do not check it
             return subject_location
 
+        if 'file' in cleaned_data and not hasattr(cleaned_data['file'], 'instance'):
+            # if new image was uploaded restore subject_location to center of new image
+            return None
```

Comment:
To do this, you'd need to return an empty string, not `None`, because the field is a `NOT NULL` field at the database level.

Reply:
However having tested this, following returning an empty string there is still a test failure;

```
======================================================================
FAIL: test_image_subject_location (tests.test_admin.FilerAdminContextTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/mwalker/apps/django-filer/tests/test_admin.py", line 1203, in test_image_subject_location
    do_test_image_subject_location(subject_location=subject_location)
  File "/Users/mwalker/apps/django-filer/tests/test_admin.py", line 1195, in do_test_image_subject_location
    self.assertEqual(saved_image.subject_location,
AssertionError: '' != '10,10'
```